### PR TITLE
uploaded dummy.txt in new test branch for example

### PR DESCRIPTION
**Pull request by geoderekh**
make sure you use the proper naing convention for your branch (i didnt)

### CODE REVIEW

**Comment by KellyVance** [comment]
**This Pull Request will be used for the demonstration this afternoon. No approval needed yet!**

**Comment by KellyVance** [comment]
This needs updating. But the basic content review Guide is included in [HERE](https://github.com/geological-survey-of-queensland/vocabularies/wiki/Vocabulary-Review-Workflow)

File: dummy.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sFNAWGHWE;OUGAE;KJ 
+GLIKOISN.LFNA;
```
Comment (KellyVance) [inline dummy.txt:2]: I think sFNAWGHWE needs to be all caps

File: dummy.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+ILVLKJlknaeinldn
```
Comment (KellyVance) [inline dummy.txt:3]:
```suggestion
skos:definition "a defintion"@en ;
```

File: dummy.txt (hunk 1 above)
Comment (geoderekh) [reply dummy.txt:2]: does it even matter vk??

Comment (KellyVance) [reply dummy.txt:2]: No probably not.

**Comment by KellyVance** [comment]
more comments

File: dummy.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+<http://linked.data.gov.au/def/australian-states-and-territories> a owl:Ontology,
+        skos:ConceptScheme ;
+    dct:created "2020-03-25"^^xsd:date ;
+    dct:creator <http://linked.data.gov.au/org/gsq> ;
+    dct:modified "2020-03-25"^^xsd:date ;
+    dct:publisher <http://linked.data.gov.au/org/gsq> ;
+    dct:source "Derived from  ISO standard 3166. https://www.iso.org/obp/ui/#iso:code:3166:AU" ;
+    skos:definition "The States and Territories of the Country of Australia."@en ;
```
Comment (KellyVance) [inline dummy.txt:16]:
```suggestion
    skos:definition "States and Territories of Australia."@en ;
```

**Comment by KellyVance** [comment]
> I agree

Additionally the three dots in the top right of each comment let you edit and quote/reply to comments. The smiley face lets you like or react to a comment (if you really feel the need).

**Comment by KellyVance** [comment]
Cancelling this before someone accidentally merges it

**Pull request closed without merging**